pkg/converter/pod_converter.go

```diff
@@ -17,11 +17,11 @@ package converter
 import (
 	"fmt"
 
-	"github.com/projectcalico/libcalico-go/lib/api"
+	api "github.com/projectcalico/libcalico-go/lib/apis/v2"
 	"github.com/projectcalico/libcalico-go/lib/backend/k8s"
 	backendConverter "github.com/projectcalico/libcalico-go/lib/converter"
 	log "github.com/sirupsen/logrus"
-	k8sApiV1 "k8s.io/client-go/pkg/api/v1"
+	k8sApiV1 "k8s.io/api/core/v1"
```

Comment:
after the dust settles, would be good to standardize our import aliases, e.g. `k8sapi` is used elsewhere

Reply:
much agreed